### ISSUE TITLE
Add VEX as a property under security-artifacts 

### ISSUE THREAD

**Issue by Danajoyluck** [ossf/security-insights-spec#84]
when a project generates VEX feed for vulnerabilities that are not exploitable, SECURITY_INSIGHTS.yml is an ideal place to capture  this information. The work around is to add VEX statement information under “security-artifacts” > “other-artifacts”. 

Having VEX as an explicit property  will make it a deterministic property for policy engines to pick it up and make decisions during software ingestion or scanners to reduce false positives.

**Comment by puerco** [comment]
We're defining a well known location in the repositories, the location could default to whatever resolves this issue:

https://github.com/openvex/spec/issues/46

**Comment by eddie-knight** [comment]
I've drafted this in #96:

```yaml
project:
  release:
    latest:
      provenance:
        vex-data: https://foo.bar/vex
```

**Comment by eddie-knight** [comment]
Thanks for this suggestion @Danajoyluck!

As per a design proposed by @puerco, support for vex is implicit within the [attestation objects](https://github.com/ossf/security-insights-spec/blob/main/specification-details/aliases.md#attestation) that will be included in the upcoming release (#97)
